frontend/packages/design-system/src/themes/Checkbox.ts

```diff
@@ -0,0 +1,41 @@
+import textTheme from './Text';
+
+export default {
+  baseStyle: {
+    backgroundColor: 'white',
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderColor: 'neutral.300',
+    borderRadius: 'nrc_xs',
+    color: 'neutral.500',
+    padding: 0,
+    _text: {
+      ...textTheme.variants.body2,
+      _web: {
+        ...textTheme.variants.body2,
+      },
```

Comment:
This seems redundant seen as they're set above

Reply:
oh yeah, this is a left over, will remove it